src/main/drivers/barometer/barometer_ms5611.c

```diff
@@ -23,7 +23,7 @@
 
 #include "platform.h"
 
-#if defined(USE_BARO) && (defined(USE_BARO_MS5611) || defined(USE_BARO_SPI_MS5611))
+#if defined(USE_BARO_MS5611)
```

Comment:
Please see my previous comment. Not a problem at the moment since only two targets using the SPI variant bat always in combination with I2C variant.